zebra/kernel_netlink.c

```diff
@@ -667,6 +682,58 @@ void nl_attr_rtnh_end(struct nlmsghdr *n, struct rtnexthop *rtnh)
 	rtnh->rtnh_len = (uint8_t *)NLMSG_TAIL(n) - (uint8_t *)rtnh;
 }
 
+int nl_rta_put(struct rtattr *rta, unsigned int maxlen, int type,
+	       const void *data, int alen)
+{
+	struct rtattr *subrta;
+	int len = RTA_LENGTH(alen);
+
+	if (RTA_ALIGN(rta->rta_len) + RTA_ALIGN(len) > maxlen) {
+		zlog_debug(
+			"%s rta_addattr_l: Error! max allowed bound %d exceeded",
```

Comment:
what does `rta_addattr_l` mean here? Given the severity, shouldn't this be a `zlog_warn` or `zlog_error`?